dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/Trn/OfficialName.cshtml.cs

```diff
@@ -70,4 +71,15 @@ public enum PreviousName
         False,
         PreferNotToSay
     }
+
+    private void SetDefaultInputValues()
+    {
+        OfficialFirstName ??= HttpContext.GetAuthenticationState().OfficialFirstName;
+        OfficialLastName ??= HttpContext.GetAuthenticationState().OfficialLastName;
+        PreviousOfficialFirstName ??= HttpContext.GetAuthenticationState().PreviousOfficialFirstName;
+        PreviousOfficialLastName ??= HttpContext.GetAuthenticationState().PreviousOfficialLastName;
+
+        HasPreviousName ??= !string.IsNullOrEmpty(PreviousOfficialFirstName) ||
+                          !string.IsNullOrEmpty(PreviousOfficialLastName) ? PreviousName.True : null;
```

Comment:
Because there are two options for 'No' (No, Prefer Not to Say) we cannot set the correct default on the radio buttons. We can only select 'Yes' if we find a previous official last name has already been set

Reply:
I'll create a separate ticket to stash `PreviousName` in `AuthenticationState` so we can populate this